arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 7
 content_type: how-to
 ---
 
-import PublicPreviewBannerPartial from '../../partials/_public-preview-banner-partial.md';
+import PublicPreviewBannerPartial from '../../partials/\_public-preview-banner-partial.md';
```

Comment:
I think that `\` excaping character is not needed here. Can you verify that your formatter is using the config from this repo (and not something else?)